alephium/scripts/09_update_guardian_set.ts

```diff
@@ -0,0 +1,24 @@
+import { Project } from '@alephium/web3'
+import { Configuration, Deployer, DeployFunction, Network } from '@alephium/cli/types'
+import { Settings } from '../alephium.config'
+
+const updateGuardianSet: DeployFunction<Settings> = async (
```

Comment:
It is only for devnet. Sometimes we have to run multiple guardians, which means that the initial fields of the `Governance` contract are different, and the contract address will also be different. To avoid modifying configuration files, I added this script to upgrade the guardian set, then the contract address will not change.

Reply:
I see. We could also deploy the bridge multiple times instead of modifying the configuration files, but I am not sure it's convenient to do so.